www/adblock2privoxy/Portfile

```diff
@@ -40,9 +40,19 @@ depends_run-append \
                     port:nginx \
                     port:privoxy
 
-variant initialize \
-    description {Initialize all configuration files. Existing
-        configurations files are not overwritten by default.} {}
+variant initialize_always \
```

Comment:
Is it really necessary to change the variant name here ? Doesn’t really seem to me to be required and will just complicate things for users who had the old variant active.

Reply:
I changed this for clarity about what the variant actually does, following up on some good feedback from @ryandesign at https://github.com/macports/macports-ports/pull/4978#discussion_r341107939

The variant is used for factory reset, and shouldn’t be used for deployed instances, as it says when used, so I don’t think the name change is a big deal.

I’ve used this approach in a few ports where the focus is configuring a service for a specific task. I’m open to other ways of accomplishing this if you can suggest one, but have found that this variant approach is simple and works.

Reply:
OK. Not sure it really makes a significant difference, but if its a variant no one would use in general then fine.